src/media/js/main.js

```diff
@@ -144,7 +144,9 @@ require(
 
     window.addEventListener(
         'resize',
-        _.debounce(function() {z.doc.trigger('saferesize');}, 200),
+        _.debounce(function() {
+            document.dispatchEvent(new CustomEvent('saferesize'));
```

Comment:
Which we only seem to require in `settings_local`. That should probably be in init also.